react/features/remote-video-menu/components/web/RemoteVideoMenuTriggerButton.js

```diff
@@ -15,6 +15,7 @@ import {
     RemoteVideoMenu,
     VolumeSlider
 } from './';
+import MuteEveryoneElseButton from './MuteEveryoneElseButton';
```

Comment:
why not import this button like the others above?

Reply:
ty!